public/apps/configuration/panels/tenant-list/tenant-list.tsx

```diff
@@ -130,24 +134,32 @@ export function TenantList(props: AppDependencies) {
     }
   };
 
-  const viewDashboard = async (tenantName: string) => {
+  const viewDashboard = async (tenantValue: string, action: string) => {
     try {
-      await changeTenant(tenantName);
+      await changeTenant(tenantValue);
       window.location.href = getNavLinkById(props.coreStart, PageId.dashboardId);
     } catch (e) {
       console.log(e);
-      addToast(createUnknownErrorToast('viewDashboard', `view dashboard for ${tenantName} tenant`));
+      addToast(
+        createUnknownErrorToast(
+          'viewDashboard',
```

Comment:
Should id also be based on action?

Reply:
Right. changed id to be based on action.